common/thorhelper/thorcommon.cpp

```diff
@@ -42,6 +42,8 @@
 
 #define ROWAGG_PERROWOVERHEAD (sizeof(AggregateRowBuilder))
 
+defaultFlushTimerInterval = nanosec_to_cycle(1000000000); // Cycles
```

Comment:
Needs to be:

    const cycle_t defaultFlushTimerInterval = nanosec_to_cycle(1000000000); // Cycles